.pre-commit-config.yaml

```
@@ -185,4 +185,9 @@ repos:
       ^type of expression at "float number value" must be number
       but found type string$
 
+- repo: https://github.com/koalaman/shellcheck-precommit
+  rev: v0.9.0
+  hooks:
+  - id: shellcheck
```

So I realized that I was using this other implementation, elsewhere…
```suggestion
- repo: https://github.com/shellcheck-py/shellcheck-py.git
  rev: v0.10.0.1
  hooks:
  - id: shellcheck
```
Let's go with it!